opencga-catalog/src/main/java/org/opencb/opencga/catalog/managers/FileManager.java

```diff
@@ -3443,6 +3523,10 @@ private OpenCGAResult<File> privateLink(String organizationId, Study study, File
                 params.setPath(params.getPath() + "/");
             }
         }
+        if (isResourcesPath(params.getPath())) {
+            throw new CatalogException("Linked files cannot be stored in the RESOURCES folder.");
```

Comment:
a Resource file can be uploaded but not linked? Why?

Reply:
Because RESOURCE files are not external by definition.